TypeCobol.Analysis.Test/BasicCfgInstrs/PerformProcedure2.dot

```diff
@@ -0,0 +1,133 @@
+﻿digraph Cfg {
+compound=true;
+node [
+shape = "record"
+]
+
+edge [
+arrowtail = "empty"
+]
+Block0 [
+label = "{START|}"
+]
+Block1 [
+label = "{MAINLINE. Block1|}"
+]
+Block2 [
+label = "{Block2|    PERFORM A-STARTUP\l}"
+]
+subgraph cluster_1{
+color = blue;
+label = "A-STARTUP";
+Block25 [
+label = "{A-STARTUP. Block25|    OPEN INPUT USER-FILE\l}"
+]
+
+}
+Block2 -> Block25 [style=dashed]
+
+Block3 [
+label = "{Block3|}"
+]
+Block4 [
+label = "{Block4|    PERFORM B-PROCESS\l}"
+]
+subgraph cluster_2{
+color = blue;
+label = "B-PROCESS";
+Block26 [
+label = "{B-PROCESS. Block26|    READ USER-FILE\l}"
+]
+Block27 [
+label = "{Block27|        AT END\l           MOVE \"Y\" TO EOF-SEEN\l}"
+]
+Block28 [
+label = "{Block28|}"
+]
+Block29 [
+label = "{B-PROCESS. Block29|}"
+]
+Block41 [
+label = "{B-LOOP. Block41|    DISPLAY \"THE ID IS \" USER-ID\l}"
+]
+Block42 [
+label = "{B-LOOP. Block42|    DISPLAY \"THE NAME IS \" USER-NAME\l}"
+]
+Block43 [
+label = "{B-LOOP. Block43|    DISPLAY \" \"\l}"
+]
+Block44 [
+label = "{B-LOOP. Block44|    DISPLAY \"ENTER TO CONTINUE\"\l}"
+]
+Block45 [
+label = "{B-LOOP. Block45|    ACCEPT ANSWER\l}"
+]
+Block46 [
+label = "{B-LOOP. Block46|    READ USER-FILE\l}"
+]
+Block47 [
+label = "{Block47|        AT END\l           MOVE \"Y\" TO EOF-SEEN\l}"
+]
+Block48 [
+label = "{Block48|}"
+]
+Block30 [
+label = "{Block30|    PERFORM B-LOOP          WITH TEST AFTER UNTIL EOF-SEEN = \"Y\"\l}"
+]
+subgraph cluster_4{
+color = blue;
+label = "B-LOOP";
+
+}
+Block30 -> Block41 [style=dashed]
+
+Block31 [
+label = "{Block31|}"
+]
+Block26 -> Block27
+Block26 -> Block28
+Block27 -> Block28
+Block28 -> Block29
+Block29 -> Block41
```

Comment:
Same problem here, the block29 should point to block30

Reply:
This is not bug but a feature 😆 ! The difference between PerformProcedure0 and PerformProcedure2 is that the `PERFORM B-LOOP` instruction uses a `TEST AFTER` clause. This PR introduces specifically some change to represent performs having a TEST AFTER clause.